easybuild/easyconfigs/o/oceanspy/oceanspy-0.2.0-foss-2022a.eb

```diff
@@ -0,0 +1,40 @@
+# Author: J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'PythonBundle'
+
+name = 'oceanspy'
+version = '0.2.0'
+
+homepage = 'https://github.com/hainegroup/oceanspy'
+description = """OceanSpy - A Python package to facilitate ocean model data analysis 
+and visualization.""" 
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('dask', '2022.10.0'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('xarray', '2022.9.0', {
```

Comment:
Please use dependency on `xarray` easyconfig

Reply:
Done. Again thanks for finding that.

Reply:
Thanks, I had to downgrade the version of `xarray` because there are already other easyconfigs depending on it this toolchain. Test should be fine now.